assets/stylesheets/shared/functions/_overflow-gradient.scss

```diff
@@ -11,7 +11,7 @@
 //
 // Usage:
 // background: overflow-gradient( var( --sidebar-rgba-test ) );
-// border-image: overflow-gradient( hex-to-rgb( $gray-light ), to left );
+// border-image: overflow-gradient( var( --color-neutral-light ) ), to left );
```

Comment:
did you mean to remove the hex-to-rgb part?

Reply:
yeah, it would need to be `-rgb`. `overflow-gradient` is [one of our functions](https://github.com/Automattic/wp-calypso/blob/master/assets/stylesheets/shared/functions/_overflow-gradient.scss#L19) and expects an rgb triplet. Unfortunately, we can't run a sass function against a custom property.

Reply:
```suggestion
// border-image: overflow-gradient( var( --color-neutral-light-rgb ) ), to left );
```